angular_components/lib/material_select/_mixins.scss

```diff
@@ -160,5 +151,23 @@
 @mixin material-select-dropdown-padding($padding) {
   ::ng-deep material-list {
     padding: $padding;
+
+    &:focus {
+      // This is a workaround for scrolling bug:
+      // https://bugs.chromium.org/p/chromium/issues/detail?id=909097
```

Comment:
FYI, this link doesn't seem to be externally visible.

Reply:
I think it should be, but maybe requires some registration with the chromium project.

Reply:
It looks like they restricted it. Maybe because a Googler filed it? It didn't have anything in it that was restricted. We can remove on the sync, but it might be useful to just see there is a bug associated with it.